### PR TITLE
Included githubactions in dependabot config

### PR DESCRIPTION
**Pull request by naveensrinivasan**
**What this PR does / why we need it**:

- Included githubactions in the dependabot config

This should help with keeping the GitHub actions updated on new releases. This will also help with keeping it secure.

Dependabot helps in keeping the supply chain secure https://docs.github.com/en/code-security/dependabot

GitHub actions up to date https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot

https://github.com/ossf/scorecard/blob/main/docs/checks.md#dependency-update-tool

Signed-off-by: naveensrinivasan <172697+naveensrinivasan@users.noreply.github.com>

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by naveensrinivasan** [comment]
A similar one was merged in another k8s repo https://github.com/kubernetes/minikube/pull/13770

**Comment by naveensrinivasan** [comment]
@olemarkus  👀  please. Thanks

**Comment by hakman** [comment]
@naveensrinivasan I appreciate the contribution, but at the moment using dependabot is not usable in the kOps repo due to heavy bazel usage. Also, at least k8s deps should be excluded.
/hold

**Comment by hakman** [comment]
Got this wrong, it's only about github actions, not gomod (yet).
/hold cancel

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+- package-ecosystem: "github-actions"
+  directory: "/"
+  schedule:
+      interval: "weekly"
```
Comment (hakman) [inline .github/dependabot.yml:6]:
```suggestion
    interval: "weekly"
    day: "sunday"
  labels:
  - "ok-to-test"
```

**Review by hakman — CHANGES_REQUESTED** [review]

**Comment by naveensrinivasan** [comment]
Thanks for the suggestion.

**Review by olemarkus — APPROVED** [review]

**Pull request merged**